selection.go

```diff
@@ -85,6 +68,12 @@ func (s *controllingSelector) nominatePair(pair *candidatePair) {
 
 func (s *controllingSelector) HandleBindingRequest(m *stun.Message, local, remote *Candidate) {
 	s.agent.sendBindingSuccess(m, local, remote)
+
+	p := s.agent.findValidPair(local, remote)
+	if p != nil && s.nominatedPair == nil && s.agent.selectedPair == nil {
```

Comment:
@hugoArregui Feel free to revert, but I think this is the best thing to do for now (and we can keep the change small/test it well)

We can follow up and make the timeout configurable (and allow the user to configure it!) but for now choosing the first candidate is best (IMO) connection startup time is really important IMO for the default.

Reply:
ok, that makes sense. My main concern is, it seems to me (but maybe this won't happen in reallity), that we could end up with a relay candidate instead of a host or a reflexive candidate. That will be bad because we have to pay for the use of the TURN server :joy: 

Otherwise I'm ok with selecting the first pair, but it seems to be we need to reintroduce the timeout right>

Reply:
Yea I agree with everything here! My vote is a follow up PR (people are hitting this bug today, so want to get it out)

In that PR I think we can implement
* Setting the connection to failed after `n` time in checking
* Allow setting the `Nomination Policy` and we will collect `validPairs` for that long. However if we get a pairing with the lowest possible priority (`host <-> host`) we should take it right away!
* This `Nomination Policy` can be configured via `webrtc.SettingEngine`
* I would be curious to know what Chromium's time for this is. It feels very fast usually.